<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditListDialog</class>
 <widget class="QDialog" name="EditListDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(239, 239, 239);</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>381</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Nom de la liste : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="styleSheet">
         <string notr="true">background-color: #FFF;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Type de liste :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #FFF;</string>
        </property>
        <item>
         <property name="text">
          <string>Alphanumérique</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Numérique</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Date</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Autre</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="styleSheet">
       <string notr="true">background-color: #FFF;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="styleSheet">
         <string notr="true">QPushButton {
padding:4px;
border:solid;
color: white;
font-size: 12px;
font-weight: 400;
background: #34495e;
}

QPushButton:pressed {
background: #2D3F51;
color: white;
}

QPushButton:hover {
background: #1C2E40;
color: white;
}</string>
        </property>
        <property name="text">
         <string>Enregistrer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="styleSheet">
         <string notr="true">QPushButton {
padding:4px;
border:solid;
color: white;
font-size: 12px;
font-weight: 400;
background: #34495e;
}

QPushButton:pressed {
background: #2D3F51;
color: white;
}

QPushButton:hover {
background: #1C2E40;
color: white;
}</string>
        </property>
        <property name="text">
         <string>Annuler</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
